docs/http/identity.md

```diff
@@ -133,7 +105,7 @@ Returns JSON in the form:
 
 The `addresses` value is an array of zero or more strings, each of which is the address of a database owned by the identity passed as a parameter.
 
-## `/identity/:identity/verify GET`
+## `GET /v1/identity/:identity/verify`
```

Comment:
In hindsight, we should've made this route instead just be `GET /v1/identity/verify`, take the auth header, and return the `Identity` that it auths as. In the new OIDC-based auth scheme, it's not trivial to derive the `Identity` from the token, and users may prefer to send requests to SpacetimeDB to do so rather than doing it themselves. Then again, maybe we'd design a different interface, like one where you just send `sub`/`iss` pairs without needing a signed token, or something. I don't know, we'll probably have to do some design work around this. No change required to this PR for this comment.